File: pgm/ABC_Dart/ABC_431_A.dart
import 'dart:io';
import 'dart:math';

void main() {
  final [int H, int B] = stdin
      .readLineSync()!
      .split(" ")
      .map(int.parse)
      .toList();
  print(max(H, B) - B);
  exit(0);
}

File: pgm/ABC_C#/ABC_431_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const int one = 1;
        const int zero = 0;
        int H = int.Parse(arr[zero]);
        int B = int.Parse(arr[one]);
        Console.WriteLine(Math.Max(H, B) - B);
        Environment.Exit(0);
    }
}